src/movableCoord.js

```diff
@@ -816,6 +817,41 @@ eg.module("movableCoord", [eg, window, "Hammer"], function(ns, global, HM) {
 		},
 
 		/**
+		 * Enables input devices
+		 * @ko 입력 장치를 사용할 수 있게 한다
+		 * @method eg.MovableCoord#enableInput
+		 * @param {HTMLElement|String|jQuery} [element] An element from which the eg.MovableCoord module is using<ko>eg.MovableCoord 모듈을 사용하는 엘리먼트</ko>
```

Comment:
Declare when element is not given.
ex) Target element or dismiss to apply for all binded elements.